internal/userutils/testhelpers/gpasswd.go

```diff
@@ -0,0 +1,62 @@
+// This is a small wrapper around the gpasswd command, which allows to run
+// gpasswd on a group file that's not at /etc/group. It does this by:
+//   - Reading the path to the group file from the GROUP_FILE environment variable.
+//   - Running the gpasswd command in a bubblewrap sandbox with the group file
+//     bind-mounted to /etc/group.
+//
+// The group file must be named "group".
+
+//go:build userutils_testhelpers
```

Comment:
Ah, also add the comment to avoid Tiobe analysis. I don't think we care adding coverage to this one, so we can likely skip it. Or just see how we use `testutils.CoverDirForTests()` and `AppendCovEnv` for other integration tests, if we want to keep high the coverage percentage :)